src/clib/cool1d_multi_g-cpp.C

```diff
@@ -0,0 +1,1740 @@
+// See LICENSE file for license and copyright information
+
+/// @file cool1d_multi_g-cpp.C
+/// @brief Declares signature of cool1d_multi_g
+
+// This file was initially generated automatically during conversion of the
+// cool1d_multi_g function from FORTRAN to C++
+
+#include <cstdio>
+#include <vector>
+#include <iostream>
+
+#include "cool1d_multi_g-cpp.h"
+#include "grackle.h"
+#include "fortran_func_decls.h"
+#include "utils-cpp.hpp"
+
+void cool1d_multi_g(
```

Comment:
Related to the earlier namespace suggestion I made:

```suggestion
void grackle::impl::cool1d_multi_g(
```